assets/scss/app.scss

```diff
@@ -346,11 +346,6 @@ textarea {
   padding-bottom: 1rem;
 }
 
-.govuk-button {
-  margin-right: 0.5rem;
-  margin-top: 30px;
-}
-
```

Comment:
Could you add a note in your pr description as to why the was removed for future reference?